website/documentation/editors/mapping-model.md

```diff
@@ -16,10 +17,122 @@ This section details the following topics:
 
 [Defining an Information Model Mapping](./information-model-mapping.html)  
 
+
 ## Model Mapping
 
 Vorto allows the user to define mapping rules to map Vorto models to other target platform domain models.
 
+The primary purpose of defining model mappings in Vorto is to directly translate or map datatype/functionblock/information model descriptions from Vorto to any other IoT platforms (for example, Kura, Eclipse Smarthome). Vorto model mapping acts as a bridge between Vorto definitions and target platforms/technologies.
```

Comment:
@nagavijays Please provide an illustration indicating the purpose of mappings in Vorto.